C/Tarefa/2/sensors.c

```diff
@@ -16,7 +18,25 @@ int get_value() {
 
 void send_data(uint8_t data) {
     // Desfaçam o empacotamento e imprimam
+    int valores[8];
+    
+     for (int i=0;i<8;i++){

```

Comment:
Costuma se colocar mais espaços para claridade: `for (int i = 0; i < 8; i++) {`. E a tabulação está estranha (passou de 4 para 5 espaços nessa linha).